private/pkg/app/appcmd/validate.go

```diff
@@ -0,0 +1,40 @@
+// Copyright 2020-2022 Buf Technologies, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package appcmd
+
+import (
+	"fmt"
+	"strings"
+)
+
+func ValidateRemoteNotEmpty(remote string) error {
+	if remote == "" {
+		return NewInvalidArgumentError("you must specify a remote module")
```

Comment:
I think this is referring to a remote address, not a remote module?

Reply:
It seems that way yeah, but the previous code all used this wording so I went with it.  Not sure which is correct.